src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -153,7 +153,11 @@ void addBlockEntity(CraftingBlockEntity te) {
         } else if (te.isStatus()) {
             this.status.add((CraftingMonitorBlockEntity) te);
         } else if (te.isAccelerator()) {
```

Comment:
The else here prevents a CPU from acting both as a storage and an accelerator, which seems like an interesting feature for addons?